### DISCUSSION TITLE
Default Hour and Minute in igx-time-picker dialog

### DISCUSSION THREAD

**Discussion by ammar4568** [IgniteUI/igniteui-angular#12849] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/19236063/229706940-05a1d4c5-d2a6-42bd-8348-bd59628c6e27.png)
Please refer to above image. When opened in dialog mode the default hours and minutes are 12 and 00 respectively. I want to change this to 07:00. Is there any option available?

**Comment by jackofdiamond5** [comment, marked as answer]
Hi @ammar4568 by default the time picker will always go to 12:00 AM. The easiest way to have it scroll to a value that you prefer is to bind it to a date object that has the required time specified.
